projects/GKTests/UITests/MainSDITests.cs

```diff
@@ -759,6 +761,65 @@ private void PersonEditDlg_Handler(PersonEditDlg dlg)
         }
 
         #endregion
+
+        public void ResearchEditDlg_Handler(ResearchEditDlg dlg)

```

Comment:
It seems that this handler, like similar ones, was moved to a separate file in order to be disbanded in the future for a set of independent tests only for this dialog - in isolation from MainSDITests. But I'm not sure, so so be it.